client/src/components/PacmanGame/gameCore.js

```diff
@@ -146,34 +146,33 @@ export const getRandomAdjacentAvailableCell = (gridState, currentLocation) => {
   return getRandomAdjacentAvailableCell(gridState, currentLocation);
 };
 
-const getGridwithWeights = (grid) => {
+export const getGridwithWeights = (grid) => {
   const gridwithWeights = grid.map((array) => {
     const newArray = array.map((element) => {
       let weight = element;
+      if (weight !== 4) weight = 0;
       if (weight === 4) weight = 1;
-      if (weight !== 1) weight = 0;
       return weight;
     });
     return newArray;
   });
   return gridwithWeights;
 };
 
-export const moveBlinky = (gridState, blinky, pacman) => {
-  const { x1, y1 } = blinky;
-  const { x, y } = pacman;
-  const gridwithWeights = getGridwithWeights(gridState);
+export const chaseLocation = (gridwithWeights, ghost, location) => {
```

Comment:
`location` seems general. Can use `currentGhostPositon` and `ghostTargetPosition`